### DISCUSSION TITLE
请教关于单独处理币安的代理

### DISCUSSION THREAD

**Discussion by shuiliu11** [XTLS/Xray-core#3911] [Q&A]
场景是这样的，XRAY开机就开着，因为日常使用中有太多需要代理了。
但是币安说我的IP是US的，只能允许我操作BINANCE US版本的。所以我把binance.com加入了直连，但是没有用。

请教有没有什么方法，就像把币安隔离，只要这个程序打开，它的所有连接就都不经过代理。

**Comment by man2018** [comment]
分流之后使用中国IP出站可以解决，我在这样子操作

**Reply by shuiliu11** [reply]
能分享下配置吗，我本身IP就是国内的，我设置：domain:binance.com直连就是国内IP，我看客户端日志里它也变成了直连，打开软件还是显示IP是US。

**Reply by xfrose** [reply]
> 能分享下配置吗，我本身IP就是国内的，我设置：domain:binance.com直连就是国内IP，我看客户端日志里它也变成了直连，打开软件还是显示IP是US。

可以试试用`geosite:binance`而不是`domain:binance.com`

**Reply by shuiliu11** [reply]
> > 能分享下配置吗，我本身IP就是国内的，我设置：domain:binance.com直连就是国内IP，我看客户端日志里它也变成了直连，打开软件还是显示IP是US。
> 
> 可以试试用`geosite:binance`而不是`domain:binance.com`

试过了，没有用。
